dev-requirements.txt

```diff
@@ -3,6 +3,8 @@ nose-exclude==0.4.1
 mock==1.3.0
 coverage==3.7.1
 tox==2.1.1
+pathlib
+reFILE
```

Comment:
What's wrong with just using mv? e.g. `for name in *; do mv "$name" ".coverage.$name"; done`?